python/jupytergis_qgis/src/plugins.ts

```diff
@@ -302,8 +302,6 @@ const activate = async (
       });
     }
   }
-
-  console.log('@jupytergis/jupytergis-qgis:qgisplugin is activated!');
```

Comment:
Should we keep these as `console.debug`? I figured it wasn't important to print this information, but I could be missing the point :)

Reply:
Yes, I think `console.debug` is better in this case. But we should make sure that we document something mentioning about to check "Verbose" option, maybe in the contributing docs.